### PR TITLE
Task/db playground 2

### PR DESCRIPTION
**Pull request by dgboss**
# Test Links:
[Landing Page](https://wps-pr-2962.apps.silver.devops.gov.bc.ca/)
[MoreCast 2.0](https://wps-pr-2962.apps.silver.devops.gov.bc.ca/morecast-2)
[Percentile Calculator](https://wps-pr-2962.apps.silver.devops.gov.bc.ca/percentile-calculator)
[MoreCast](https://wps-pr-2962.apps.silver.devops.gov.bc.ca/morecast)
[C-Haines](https://wps-pr-2962.apps.silver.devops.gov.bc.ca/c-haines)
[FireBat](https://wps-pr-2962.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator)
[FireBat bookmark](https://wps-pr-2962.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator?s=266&f=c5&c=NaN&w=20,s=286&f=c7&c=NaN&w=16,s=1055&f=c7&c=NaN&w=NaN,s=305&f=c7&c=NaN&w=NaN,s=344&f=c5&c=NaN&w=NaN,s=346&f=c7&c=NaN&w=NaN,s=328&f=c7&c=NaN&w=NaN,s=1399&f=c7&c=NaN&w=NaN,s=334&f=c7&c=NaN&w=NaN,s=1082&f=c3&c=NaN&w=NaN,s=388&f=c7&c=NaN&w=NaN,s=309&f=c7&c=NaN&w=16,s=306&f=c7&c=NaN&w=NaN,s=1029&f=c7&c=NaN&w=NaN,s=298&f=c7&c=NaN&w=NaN,s=836&f=c7&c=NaN&w=NaN,s=9999&f=c7&c=NaN&w=NaN)
[Auto Spatial Advisory (ASA)](https://wps-pr-2962.apps.silver.devops.gov.bc.ca/auto-spatial-advisory)
[HFI Calculator](https://wps-pr-2962.apps.silver.devops.gov.bc.ca/hfi-calculator)

### CODE REVIEW

**Comment by conbrad** [comment]
Closing to save resources

**Review by conbrad — APPROVED** [review]

File: .github/workflows/deployment.yml [hunk 1]
```diff
@@ -22,7 +22,7 @@ jobs:
         shell: bash
         run: |
           oc login "${{ secrets.OPENSHIFT_CLUSTER }}" --token="${{ secrets.OC4_DEV_TOKEN }}"
-          EPHEMERAL_STORAGE=True bash openshift/scripts/oc_provision_db.sh ${SUFFIX} apply
+          EPHEMERAL_STORAGE=False bash openshift/scripts/oc_provision_db.sh ${SUFFIX} apply
```
Comment (conbrad) [inline .github/workflows/deployment.yml:25]: Why this change?

**Review by conbrad — COMMENTED** [review]
Summary: Looks good!

File: .github/workflows/deployment.yml (hunk 1 above)
Comment (dgboss) [reply .github/workflows/deployment.yml:25]: Yeah, I was going to reach out to you about this and the changes to PVC sizes. In order to conserve resources we don't allocate 'real' PVCs for dev deployments and instead rely on transient storage of some sort (I think). In order to test this PR, we needed large PVCs so that I could throw large quantities of data into the PR patroni instance and create the materialized views. 

I'll roll back this change and revert the changes to openshift/templates/patroni.yaml and openshift/templates/patroni_prerequisite.yaml.

Comment (conbrad) [reply .github/workflows/deployment.yml:25]: All good makes sense, this comment was mostly me showing @brettedw the code review process :)

File: api/app/db/crud/weather_models.py [hunk 2]
```diff
@@ -369,8 +392,7 @@ def get_latest_station_prediction_per_day(session: Session,
         .join(PredictionModel, PredictionModelRunTimestamp.prediction_model_id == PredictionModel.id)\
         .join(subquery, and_(
             WeatherStationModelPrediction.prediction_timestamp == subquery.c.latest_prediction,
-            WeatherStationModelPrediction.station_code == subquery.c.station_code))\
-        .order_by(WeatherStationModelPrediction.update_date.desc())
```
Comment (conbrad) [inline api/app/db/crud/weather_models.py:373]: I think we can undo this change since it's not the original table we're concerned about here right?

Comment (dgboss) [reply api/app/db/crud/weather_models.py:373]: Yeah, i was experimenting with performance to see if we could each out a few more milliseconds by not ordering the results. It didn't buy us more than a couple milliseconds so we can definitely revert.
